### ISSUE TITLE
Add numpy to requirement to fix MAC and Win testing errors

### ISSUE THREAD

**Issue by tashrifbillah** [pnlbwh/SlicerDiffusionQC#35]
Add numpy [here](https://github.com/pnlbwh/SlicerDiffusionQC/blob/master/SuperBuild/External_python-diffusionqclib.cmake#L18) .

[This](http://slicer.cdash.org/testDetails.php?test=9889819&build=1882912) is the error:
```bash
Traceback (most recent call last):
  File "/Volumes/Dashboards/Stable/S-4102-E-b/DiffusionQC/Test/run_test.py", line 3, in <module>
    import numpy as np
ImportError: No module named numpy
```

**Comment by tashrifbillah** [comment]
Introduced by https://github.com/pnlbwh/SlicerDiffusionQC/commit/014cb2f9689e571b6e52f9c48c91481c81d9d1e8

**Comment by tashrifbillah** [comment]
See the testing errors here:
http://slicer.cdash.org/buildSummary.php?buildid=1879552

I believe PYTHONPATH isn't set properly on MAC and Windows

**Comment by tashrifbillah** [comment]
numpy removed by https://github.com/pnlbwh/SlicerDiffusionQC/commit/cfd6bcfb64f5f064da0a53f1cfb7367a5ec61cdf

**Comment by tashrifbillah** [comment]
Fix given in issue #38
